control_msgs[] group_control